### PR TITLE
Feature/teach interface

### PR DESCRIPTION
**Pull request by gor181**
Focus of this PR is to replace old UI images and descriptions with new ones. Related to https://github.com/datacamp/teach-features/issues/61

### CODE REVIEW

**Comment by ncarchedi** [comment]
@richierocks can you own reviewing/merging this PR since you know best what will be useful for content + instructors?

**Comment by ncarchedi** [comment]
@richierocks ping

File: docs/interface/dashboard.md [hunk 1]
```diff
@@ -1,30 +1,16 @@
...
 - **Instructors**: instructors of the repository. You can set these in the Settings tab of the master branch of the repository.
-- **Updated by me**: when the repository was last updated by me
-- **Updated by instructor**: when the repository was last updated by an instructor
-- **Un reviewed time**: the time differences between when the repository was last updated by me and when it was last updated by an instructor
-- **Last updated branch**: the branch of this repository that was last updated
+- **Last edited**: time when the last update has been done to the content
+- **Edit action**: opens editor for editing the content
 
-For featured courses the following extra columns are in the table:
+For premium content some columns are redefined:
```
Comment (richierocks) [inline docs/interface/dashboard.md]: "redefined" from what?

File: docs/interface/dashboard.md [hunk 2]
```diff
@@ -1,30 +1,16 @@
...
-The following features are also important in the course dashboard:
-
-- The **search field** will do string matching in the following columns: title, owner, instructors and last updated branch.
-- Checking the **"Only my courses"** checkbox will show you only courses that are owned by you. 
-***Note**: the first time using the dashboard, no courses will be owned by you.*
-- In the upper right corner of the dashboard, you can choose between Featured/Development courses. Featured courses are courses that are in production on DataCamp.
-- You can synchronize the shown repositories with GitHub by hitting the **"Refresh List"** link in the upper right corner of the dashboard.
-- The table is paginated, which means the rows are shown per 10. You can show all the repositories, but this can be slow if there are a lot of repositories.
\ No newline at end of file
+- **Edit action**: allows advanced administration of branches and content
```
Comment (richierocks) [inline docs/interface/dashboard.md]: I think it's worth adding a few lines to explain the edit popup. Something like

> - **View** takes you to the course landing page for that branch.
> - **Edit** takes you to the Tech editor for that branch.
> - **Create new branch** allows you to create a new branch, based upon one of the existing branches.

File: docs/interface/dashboard.md [hunk 3]
```diff
@@ -1,30 +1,16 @@
...
-*Teach dashboard*
+![Teach dashboard](/images/interface/content-dashboard.png)
+*Content dashboard*
 
-The landing page consists of two parts: a sortable, filterable and paginated table of course repositories and a modal to create new courses. The table has the following columns:
+The content overview page consists of two parts: a filterable and paginated table of created content and a modal to create new or import existing courses. The table has the following columns:
 
 - **Title**: the title of the course (or repository name if it is not properly built yet)
-- **Owner**: the owner of the repository. If the repository has no owner yet, you can claim it if it's yours.
+- **Status**: current state of the content (development, live)
```
Comment (richierocks) [inline docs/interface/dashboard.md]: Might be useful to enumerate all the possible states, and what they mean. for example, the difference between Live and Published isn't totally intuitive.

File: docs/interface/create-content-dialog.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
 When creating the content you will be using the Create content dialog which consists of:
 
-- **Dropdown** where you can select on which Github account you want to create the repository from (*Useful when you have more than 1 organization connected to your account*)
 - **Name** of the course you are creating
-- **Link existing github repository** is an option where DataCamp is going to fetch the names of all your Github repositories and offer you the possibility to import those repositories as courses. (*In order to work your Github repository should contain [structure](/courses/repo-structure.md) DataCamp build process understands)
-
-Once "Create" button has been clicked DataCamp is going to create the scaffolded course on which you can continue working. We recommend checking the [Exercises page](/courses/exercises/README.md) on best way to continue authoring your content.
\ No newline at end of file
+- **Dropdown** where you can select on which Github account you want to create the repository from (*Useful when you have more than 1 organization connected to your account*)
+- **Technology** your course is going to be based on
```
Comment (richierocks) [inline docs/interface/create-content-dialog.md]: Enumerate the choices.

File: docs/interface/dashboard.md (hunk 1 above)
Comment (ncarchedi) [reply docs/interface/dashboard.md]: I don't think we should even be saying what it used to be. Let's just tell them what it is now.

File: docs/interface/dashboard.md (hunk 2 above)
Comment (ncarchedi) [reply docs/interface/dashboard.md]: 👍

File: docs/interface/dashboard.md (hunk 3 above)
Comment (ncarchedi) [reply docs/interface/dashboard.md]: Do we still have a `published` state? I think it's just `development`, `soft_launch`, and `live` now for premium courses (in that order). @gor181 please confirm

File: docs/interface/create-content-dialog.md (hunk 4 above)
Comment (ncarchedi) [reply docs/interface/create-content-dialog.md]: it's just Python, R, and SQL right now

File: docs/interface/dashboard.md (hunk 3 above)
Comment (gor181) [reply docs/interface/dashboard.md]: We fetch the state from main app and then we display it. The definition and state transitions are available at https://github.com/datacamp/main-app/blob/master/app/models/concerns/course_state_transitions.rb.

Afaik, published state is used for non premium courses in order to make them available on the permanent url.

Comment (gor181) [reply docs/interface/dashboard.md]: @mayrsascha ^ please confirm

**Comment by gor181** [comment]
@richierocks @ncarchedi - Updated as per suggestions. Demo at https://teach-documentation-pr-162.herokuapp.com/interface/dashboard.html

**Comment by ncarchedi** [comment]
I think this PR needs some work so closing it for now. Will open a new one in the next few days, before we launch the new editor to our instructors.

**Pull request closed without merging**